__tests__/app.test.js

```diff
@@ -45,3 +45,32 @@ describe("GET /api", () => {
 			});
 	});
 });
+
+describe("GET /api/articles/:article_id", () => {
+	test("should return a status code of 200 with the requested article", () => {
+		return request(app)
+			.get("/api/articles/1")
+			.expect(200)
+			.then(({ body }) => {
+				expect(typeof body.article).toBe("object");
+				expect(body.article).toHaveProperty("article_id");
+				expect(body.article).toHaveProperty("title");
+				expect(body.article).toHaveProperty("author");
+				expect(body.article).toHaveProperty("body");
+				expect(body.article).toHaveProperty("topic");
+				expect(body.article).toHaveProperty("created_at");
+				expect(body.article).toHaveProperty("votes");
+				expect(body.article).toHaveProperty("article_img_url");
+			});
```

Comment:
It might be a nie idea to check that the id that is being returned definitely matches the one that you searched for :)